src/components/Header/Header.css

```diff
@@ -0,0 +1,126 @@
+.header {
+  width: 90%;
+  height: 4rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #131a17;
+  padding: 0 10%;
+}
+
+.logo {
+  font-size: 2rem;
+  color: white;
+  font-weight: bold;
+}
+
+.navigation {
+  position: relative;
+  display: flex;
+  align-items: flex-end;
+  height: 40px;
+  width: 60%;
+  padding: 0.5rem 0rem;
+  background-color: #131a17;
+  color: black;
+  margin-left: auto;
+  font-weight: bold;
+  font-size: 1.3rem;
```

Comment:
Using 1.3rem instead of 1.25rem for font sizes or other measurements in your CSS can lead to inconsistencies and might not align with typical best practices for web design